python/pyarrow/src/arrow/python/platform.h

```diff
@@ -24,7 +24,7 @@
 // to mean Py_ssize_t (defining this to suppress deprecation warning)
 #define PY_SSIZE_T_CLEAN
 
-#include <Python.h>  // IWYU pragma: export
+#include "arrow/python/pythoncapi_compat.h"  // IWYU pragma: export
```

Comment:
We should probably put this file in a `arrow/python/vendored` subdirectory instead.

Reply:
Done.